README.md

```diff
@@ -12,6 +12,14 @@ Reward payments from bakers to delegators are a cornerstone of Tezos' delegation
 - establishing payout tool accessible to (and open to contribution from) the ecosystem of JS/TS developers.
 - delivering life improvements to bakers continuously
 
+### Quick Start
+
+To install breadcrumbs just:
+
+1. [download latest binary release](https://github.com/kalouo/breadcrumbs/releases).
+2. (linux only) enable executable flag
+   - `chmod +x bc-linux-*`
```

Comment:
This has to be run from the folder in which `breadcrumbs` is contained I presume? If so, would clarify.